webservice/soap/lib/nusoap.php

```diff
@@ -2638,7 +2638,7 @@ public function send($data, $timeout=0, $response_timeout=30, $cookies=null)
     * @access   public
     * @deprecated
     */
-    public function sendHTTPS($data, $timeout=0, $response_timeout=30, $cookies)
+    public function sendHTTPS($data, $timeout=0, $response_timeout=30, $cookies=NULL)
```

Comment:
I'm surprised that no typing was introduced here at all, that should have been done as part of the PHP 8 project. @smeyer-ilias did we forget something?

Reply:
@c-knof `NULL` must be `null` according to the code style, you can fix that with the cs-fixer: ```libs/composer/vendor/bin/php-cs-fixer fix --config=CI/PHP-CS-Fixer/code-format.php_cs --using-cache=no  webservice/soap/lib/nusoap.php ```

Reply:
@chfsx code style is fixed. Can I just insert the copyright section into this file? I was very unsure about it, so I didn't do it